lumicks/pylake/force_calibration/tests/test_driving_input.py

```diff
@@ -0,0 +1,32 @@
+from lumicks.pylake.force_calibration.detail.driving_input import estimate_driving_input_parameters
+import numpy as np
+import pytest
+
+
+@pytest.mark.parametrize(
+    "amps, frequencies, true_amp, true_freq, guess",
+    [
+        [[0.5, 1], [36.9, 50], 0.5, 36.9, 36],
+        [[0.34, 1], [36.7, 50], 0.34, 36.7, 36],
+        [[0.45, 1], [36.554, 50], 0.45, 36.554, 36],
+        [[0.5, 1], [36.9, 42.0], 0.5, 36.9, 36],  # Does window exclude the bigger peak?
+        [[0.5, 1], [36.9, 41.0], 1.0, 41.0, 36],  # Does it pick up on the second peak?
+        [[0.5, -1], [36.9, 41.0], 1.0, 41.0, 36],  # Does it pick up on the second peak?
+        [[0.5, 0], [2.4, 50], 0.5, 2.4, 3],  # Does it reject offset correctly?
+    ],
+)
+def test_driving_input_estimation(amps, frequencies, true_amp, true_freq, guess):
```

Comment:
Maybe consider using `expected_amp|freq` instead of `true_amp|freq`.

Reply:
👍